### DISCUSSION TITLE
Project templates

### DISCUSSION THREAD

**Discussion by sabuto** [AvaloniaCommunity/Prism.Avalonia#37] [Show and tell]
I have created and published the project templates, I would appreciate some feedback if anyone has the time. I have essentailly taken the avalonia project templates and adapted them for prism.avalonia. I have included the simple dialog app and the sample mvvm app. I have the version currently set to 0.0.2 I will bring it upto date with the avalonia version when I have it in a reasonable position and it has been tested more. 

[Repository](
https://github.com/sabuto/Avalonia-prism-dotnet-templates)

**Comment by DamianSuess** [comment]
Skimmed though it and it looks good so far. Can't wait to give this a test drive
